linta-android/src/main/java/com/swvl/lint/checks/DuplicateResourceFilesDetector.kt

```diff
@@ -56,39 +56,42 @@ class DuplicateResourceFilesDetector : ResourceXmlDetector() {
             .transform(DOMSource(document), StreamResult(stringWriter))
 
         currentDocument = stringWriter.toString()
+        stringWriter.flush()
```

Comment:
and this

Reply:
It's to flush the memory consumed by the string writer.

Reply:
as agreed, we'll keep it since it seems to enhance the time 👍